ios/FluentUI/Bottom Commanding/CommandingItem.swift

```diff
@@ -86,6 +95,11 @@ open class CommandingItem: NSObject {
         self.isToggleable = isToggleable
     }
 
+    @objc public convenience init(title: String, image: UIImage, trailingView: UIView, action: @escaping (CommandingItem) -> Void, isToggleable: Bool = false) {
```

Comment:
do we need this init method? the trailingView feels like a transient thing clients will update as they need to highlight something in the cell. does it make sense to have it in init if we already have a public objc var that they can easily customize? trying to be conservative with our binary impact.